wiki/fabrication/ultimaker-series.md

```diff
@@ -0,0 +1,100 @@
+---
+date: 2022-12-07
+title: Ultimaker Series (3/S3)
+---
+The Ultimaker Series of 3D pinters are a relatively beginner-friendly set of printers, with a variety of advanced print settings at the user's fingertips if they need them. This series offers easy reload of material, touch screen interface, dual extrusion of materials, and options to print with both Wireless or USB connection. Printer material options include PLA, Tough PLA, ABS, Nylon, Breakaway, PVA, and more.
+
+
+In the MS in Robotic Systems Development (MRSD) Lab at CMU, there are four printers from the Ultimaker series. These can be used in essentially the same way. The printers are:
+- 2 x Ultimaker 3 (no longer manufactured by Ultimaker)
+- 2 x Ultimaker S3
+
+
+## Software
+The Ultimaker Series of printers uses the [Ultimaker Cura](https://ultimaker.com/software/ultimaker-cura) software to slice STL files for printing. When setting up a print, it is important to crosscheck your print settings with the printer you have on hand. This is expanded on below.
+
+
+## Setting up a print
+Using your favorite CAD Sofware (i.e. Solidworks), prepare an STL file of your model for printing. Open up CURA, and use the folder icon in the top left of your window (Circle 1) to select the STL file(s) you have prepared.
+
+Using the options on the left side of the screen (Circle 2), move your piece(s) around. Using these controls, you can change the orientaiton of the part, as well as its location on the build plate. Make sure that part is oriented to minimize overhangs, and if you have multiple pieces, that the pieces are not too close to each other.
+
+<figcaption align = "center"><b>Figure 1: Ultimaker Cura PREPARE screen when preparing a print. </b></figcaption>
```

Comment:
@smbryan do you know if `<figcaption>` tags are supported by our markdown flavor?

Reply:
Please remove the HTML tag used in the caption - these tags will break our markdown renderer.